File: Desktop/SummerVacation/0703/python/numThirtyFive.py
import random
name = list()
finish = True
while finish :
    player = input("Enter a name: ")
    if len(player) == 0 :
        finish = False
    else :
        name.append(player)

if len(name) != 0 :
    print("The winner is",name[random.randrange(0,len(name))])
else :
    print("You need more player.")

File: Desktop/SummerVacation/0703/src/numThirtyFive.java
import java.util.Vector;
import java.util.Scanner;
import java.util.Random;
public class numThirtyFive {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        Vector<String> name = new Vector<String>(1);
        String player = null;
        Random choice = new Random();
        boolean finish = true;
        do{
            System.out.print("Enter a name: ");
            player = scan.nextLine();
            if(player.length() !=0){
                name.add(player);
            }
            else
                finish = false;

        }while(finish);
        if(name.size() > 0){
            int a = choice.nextInt(name.size());
            System.out.print("The winner is " + name.elementAt(a));
        }
        else
            System.out.print("Input properly.");
    }
}
